cvat-ui/src/components/email-confirmation-page/email-confirmed.tsx

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Layout, Row, Col } from 'antd';
+
+const { Content } = Layout;
+
+/**
+ * Component for displaying email confirmation message and then redirecting to the loginpage
+ */
+class EmailConfirmationMessage extends React.Component {
```

Comment:
Please look at antd Countdown component, if you use it, you don't need this lifecircle method and you don't need React ES6 components here.

Example:

```
function onFinish() {
  document.getElementById('link').click();
}

const deadline = Date.now() + 1000 * 5; 

ReactDOM.render(
  <Row gutter={16}>
    <Col span={12}>
      <Countdown format='ss' title="Redirecting to login page after.." value={deadline} onFinish={onFinish} />
    </Col>
  </Row>,
  mountNode,
);
```

Reply:
yes, i removed the class component. And with the help of Countdown component it was more east.